pubsub/test/subscriptions.test.js

```diff
@@ -188,57 +188,57 @@ describe('pubsub:subscriptions', function () {
       var sample = getSample();
       var error = 'error';
       sample.mocks.subscription.ack.callsArgWith(1, new Error(error));
-      sample.program.pull(subscriptionName, function (err) {
+      sample.program.pullMessages(subscriptionName, function (err) {
         assert(err);
         assert(err.message === 'error');
       });
     });
   });
 
-  describe('printUsage', function () {
-    it('should print usage', function () {
+  describe('main', function () {
+    it('should call createSubscription', function () {
       var program = getSample().program;
 
-      program.printUsage();
-
-      assert(console.log.calledWith('Usage: node subscriptions COMMAND [ARGS...]'));
-      assert(console.log.calledWith('\nCommands:\n'));
-      assert(console.log.calledWith('\tcreate TOPIC_NAME SUBSCRIPTION_NAME'));
-      assert(console.log.calledWith('\tdelete SUBSCRIPTION_NAME'));
-      assert(console.log.calledWith('\tpull SUBSCRIPTION_NAME'));
-      assert(console.log.calledWith('\tlist [TOPIC_NAME]'));
-      assert(console.log.calledWith('\nExamples:\n'));
-      assert(console.log.calledWith('\tnode subscriptions create my-topic my-subscription'));
-      assert(console.log.calledWith('\tnode subscriptions delete my-subscription'));
-      assert(console.log.calledWith('\tnode subscriptions pull my-subscription'));
-      assert(console.log.calledWith('\tnode subscriptions list'));
-      assert(console.log.calledWith('\tnode subscriptions list my-topic'));
+      sinon.stub(program, 'createSubscription');
+      program.main(['create', topicName, subscriptionName]);
+      assert.equal(program.createSubscription.calledOnce, true);
+      assert.deepEqual(program.createSubscription.firstCall.args.slice(0, -1), [topicName, subscriptionName]);
     });
-  });
 
-  describe('main', function () {
-    it('should call the right commands', function () {
+    it('should call deleteSubscription', function () {
       var program = getSample().program;
 
-      sinon.stub(program, 'create');
-      program.main(['create']);
-      assert(program.create.calledOnce);
+      sinon.stub(program, 'deleteSubscription');
+      program.main(['delete', subscriptionName]);
+      assert.equal(program.deleteSubscription.calledOnce, true);
+      assert.deepEqual(program.deleteSubscription.firstCall.args.slice(0, -1), [subscriptionName]);
+    });
 
-      sinon.stub(program, 'delete');
-      program.main(['delete']);
-      assert(program.delete.calledOnce);
+    it('should call listSubscriptions', function () {
+      var program = getSample().program;
+      sinon.stub(program, 'listSubscriptions');
```

Comment:
Is there any difference/preference between
```javascript
sinon.stub(program, 'listSubscriptions');
```
and
```javascript
program.listSubscriptions = sinon.stub();
```

If not, should we stick to one of these idioms?